src/templates/en/2019/base_chapter.html

```diff
@@ -24,14 +24,16 @@
 {% endmacro %}
 
 {% macro render_authors() %}
-<ul>
   {% for author in metadata.get('authors') %}
-    {% set authordata = config.contributors[author] if author in config.contributors else None %}
+  {% if loop.length == 1 %}<h4>Author</h4>{% endif -%}
+  {% if loop.length > 1 and loop.index == 1 %}<h4>Authors</h4>{% endif %}
+  {% if loop.index == 1 %}<ul>{% endif %}
+      {% set authordata = config.contributors[author] if author in config.contributors else None %}
     {% if authordata %}
       <li>
-          <img class="avatar" alt="Author name" src="{{ authordata.avatar_url if authordata.avatar_url else '/static/images/avatars/'+str(randrange(1, 12))+'.jpg'  }}">
+          <a href="https://almanac.httparchive.org{{ url_for('contributors', year=year, lang=lang, _anchor=author) }}"><img class="avatar" alt="{{ authordata.name }}" src="{{ authordata.avatar_url if authordata.avatar_url else '/static/images/avatars/'+str(randrange(1, 12))+'.jpg'  }}" /></a>
```

Comment:
I think we should do the avatar fallback URL somewhere in the py

Reply:
Not sure what you mean.

The Avatar URL? Isn't that already there with this?
`src="{{ authordata.avatar_url if authordata.avatar_url else '/static/images/avatars/'+str(randrange(1, 12))+'.jpg'  }}`
Just looked at the Accessibility Chapter in my branch (the only one with an Author without an Avatar!) and I see a fallback Avatar for Nektarios.

Or do you mean the fallback link URL? Well if the author isn't found then it will just link to the Contributors page with an unknown hash ref, and that's the page we'd probably want to fallback to isn't it? So currently that would take you to https://almanac.httparchive.org/en/2019/contributors#authorwhodoesnotexist - which is fine as will still go to contributors page and just can't scroll to the `authorwhodoesnotexist` id that doesn't exist. Or do you think we should fallback to another page? Or remove the link altogether in this situation?

Reply:
Oh sorry, I meant that the templates shouldn't have to decide what to do if there isn't an avatar_url. I see that it wasn't actually changed in this PR but I just noticed it.

For example when we assign the avatar_url a value in https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/src/config.py#L23 we should make sure that in all cases it will have either a gravatar URL, GitHub URL, or randomly-generated character URL. AFAICT in that code it should cover all of our bases so maybe we can just remove the fallback logic from the template?

```suggestion
          <a href="https://almanac.httparchive.org{{ url_for('contributors', year=year, lang=lang, _anchor=author) }}"><img class="avatar" alt="{{ authordata.name }}" src="{{ authordata.avatar_url }}" /></a>
```

Reply:
OK tested that and also worked. Pushed now!